backend/src/events/controllers.ts

```diff
@@ -280,6 +281,31 @@ const getAttendees = async (eventID: string, userID: string) => {
   });
 };
 
+const updateEnrollmentStatus = async (eventID: string, userID: string, newStatus: any) => {
+  return await prisma.eventEnrollment.update({
+    where: {
+      userId_eventId: {
+        userId: userID,
+        eventId: eventID,
+      },
+    },
+    data: {
+      attendeeStatus: newStatus
+    },
+  });
+}
+
+const getEnrollmentStatus = async (eventID: string, userID: string) => {
+  return await prisma.eventEnrollment.findUnique({
+    where: {
+      userId_eventId: {
+        userId: userID,
+        eventId: eventID,
+      },
+    },
+  });
+}
+
```

Comment:
you will not be needing this with the new changes I pushed. Please ask any questions if you have on why